File: 2015/deno/day-/part-1.ts
export function processPart1(input: string): number {
	const trimmed = input.trim();
	const directions = trimmed.split('');

	let floor = 0;
	for (const direction of directions) {
		if (direction === '(') {
			floor++;
		} else if (direction === ')') {
			floor--;
		}
	}
	return floor;
}

File: 2015/go/day01/part1.go
package main

func processPart1(input string) int {
	floor := 0
	for _, c := range input {
		if c == '(' {
			floor++
		} else if c == ')' {
			floor--
		}
	}
	return floor
}

func test() int {
	return 1
}
